CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ This document follows the conventions laid out in [Keep a CHANGELOG](https://kee
 
 ### Changed
 
+ - Update python minimum requirement from 3.8 to 3.9 ()
```

Comment:
```suggestion
 - Update python minimum requirement from 3.8 to 3.9 ([#484](https://github.com/ansys/pymechanical/pull/484))
```

Reply:
this will be update in coming commit since PR was made after first commit